### PR TITLE
[CORL-2509] update commented out tests

### PR DESCRIPTION
**Pull request by marcushaddon**
## What does this PR do?
This PR commented out updates unit tests in `src/core/client/stream/tabs/Comments/Comment/CommentContainer.spec.tsx` to integration tests using RTL (where possible).

## These changes will impact:

- [ ] commenters
- [ ] moderators
- [ ] admins
- [x] developers

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags?
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
`npm test src/core/client/stream/test/comments/components/CommentContainer.spec.tsx` and observe that all tests pass

## Where any tests migrated to React Testing Library?
Yes. The tests in `src/core/client/stream/tabs/Comments/Comment/CommentContainer.spec.tsx` (now deleted) have been migrated.

## How do we deploy this PR?
No special considerations should be needed.

### CODE REVIEW

File: src/core/client/stream/test/comments/components/CommentContainer.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
+import { screen, within } from "@testing-library/react";
+import { pureMerge } from "coral-common/utils";
```
Comment (kabeaty) [inline src/core/client/stream/test/comments/components/CommentContainer.spec.tsx:2]: Can you add a space here to group these imports?

File: src/core/client/stream/test/comments/components/CommentContainer.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+  expect(firstCommentElement).toBeInTheDocument();
+  const replyButton = within(firstCommentElement).getByLabelText("Reply", {
+    exact: false,
+  });
+  expect(replyButton).toBeInTheDocument();
+  expect(replyButton).toBeDisabled();
+});
+
+it.only("renders with tombstone when comment has been deleted", async () => {
```
Comment (kabeaty) [inline src/core/client/stream/test/comments/components/CommentContainer.spec.tsx]: Please remove this to make sure it's running all the tests :)

**Review by kabeaty — CHANGES_REQUESTED** [review]
The tests look great, yay, so happy to see them there and not commented-out!

I left a few comments throughout. Can you also remove the commented-out test file since we don't need it anymore?

File: src/core/client/stream/test/comments/components/CommentContainer.spec.tsx (hunk 1 above)
Comment (marcushaddon) [reply src/core/client/stream/test/comments/components/CommentContainer.spec.tsx:2]: ✅

File: src/core/client/stream/test/comments/components/CommentContainer.spec.tsx (hunk 2 above)
Comment (marcushaddon) [reply src/core/client/stream/test/comments/components/CommentContainer.spec.tsx]: ✅

**Review by kabeaty — APPROVED** [review]

**Pull request merged**